src/components/Text/index.tsx

```diff
@@ -4,7 +4,7 @@ import { jsx } from "@emotion/core";
 import { Theme } from "../../types/global";
 
 type TextTags = "p" | "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
-type FontSizes = 1 | 2 | 3 | 4 | 5;
+type FontSizes = 1 | 2 | 3 | 4 | 5 | 6;
 type SpaceScale = 1 | 2 | 3 | 4 | 5 | 6;
```

Comment:
we're using these same space types in a few different places, we should move these to the global types. doesnt have to be done now, but something to add to the next up